src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/ProjectHotReloadSession.cs

```diff
@@ -73,13 +73,20 @@ public async Task<bool> ApplyLaunchVariablesAsync(IDictionary<string, string> en
             }
 
             return false;
+}
+
+        // TODO: remove when Web Tools is no longer calling this method.
```

Comment:
Is there a work item tracking the WebTools side of this? Can we include the URL here?

Reply:
No particular work item, just ongoing Hot Reload work.